### PR TITLE
Tag SQS queues with DEPLOYMENT_PREFIX

### PR DESCRIPTION
**Pull request by jpopelka**
Related to https://github.com/openshiftio/openshift.io/issues/1081

### CODE REVIEW

File: hack/queue_conf.py [hunk 1]
```diff
@@ -47,10 +49,23 @@ def set_queue_attributes(queue_names):
...
+                Tags={
+                    'ENV': _DEPLOYMENT_PREFIX
+                }
+            )
+            _logger.debug("Remote responded with %r after setting queue tag for %r",
+                          response, queue_name)
+            # Tagging API actions are limited to 5 TPS (Transactions Per Second) per AWS account.
+            # http://docs.aws.amazon.com/AWSSimpleQueueService/latest/
+            # SQSDeveloperGuide/limits-queues.html
+            sleep(1 / 5)
```
Comment (msrb) [inline hack/queue_conf.py]: [This](https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/468/files#diff-8c255b087ddc69d83b81d4fb3a18c9cbR26) will run once per deployment, but we have `-ingestion`, `-api`, ... deployments - it's a mess. And they will all execute this script +/- at the same time.

Comment (jpopelka) [reply hack/queue_conf.py]: They won't since #468

Comment (jpopelka) [reply hack/queue_conf.py]: Check latest commit please.

**Comment by jpopelka** [comment]
Testing on dev:

```
INFO:__main__:Creating or requesting QueueURL for queue 'jpopelka_api_AnityaTask_v0'
DEBUG:__main__:Remote responded with {'ResponseMetadata': {'HTTPStatusCode': 200, 'RetryAttempts': 0, 'RequestId': 'b377aeb9-e9c8-5bd4-b1ee-fc0ae97156a9', 'HTTPHeaders': {'content-type': 'text/xml', 'date': 'Tue, 19 Dec 2017 11:56:41 GMT', 'connection': 'keep-alive', 'content-length': '225', 'server': 'Server', 'x-amzn-requestid': 'b377aeb9-e9c8-5bd4-b1ee-fc0ae97156a9'}}} after setting queue attributes for 'jpopelka_api_AnityaTask_v0'
Traceback (most recent call last):
  File "/usr/bin/queue_conf.py", line 91, in <module>
    set_queue_attributes(sys.argv[1].split(','))
  File "/usr/bin/queue_conf.py", line 56, in set_queue_attributes
    response = client.tag_queue(
  File "/usr/lib/python3.4/site-packages/botocore/client.py", line 551, in __getattr__
    self.__class__.__name__, item)
  AttributeError: 'SQS' object has no attribute 'tag_queue'
```

**Comment by jpopelka** [comment]
Tested on dev & works.

**Pull request merged**